g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/conf/CollectSourceValidationInterceptor.java

```diff
@@ -181,8 +183,9 @@ private static void validateLocale(HttpServletRequest request, List<String> sour
 		}
 		if (!RegExpValidatorUtils.IsLetterAndNumberAndValidchar(locale)) {
 			throw new VIPAPIException(ValidationMsg.LOCALE_NOT_VALIDE);
-		}else if (!sourceLocales2.contains(locale)) {
-			throw new VIPAPIException(ValidationMsg.LOCALE_NOT_VALIDE);
+		}
+		if (!sourceLocales2.get(0).equals(ConstantsKeys.FALSE) && !sourceLocales2.contains(locale)) {
```

Comment:
It's strange to use argument of index 0 to judge.

Reply:
the list interface not support get first method